STYLE-GUIDE.md

```diff
@@ -0,0 +1,278 @@
+* A good comment is often less effective than a good variable name.
+
+
+### Padding & additional whitespace
+TuxedoJS has a consistent convention for white space. There is a space after an `if`, `while`, `for`, `function`, etc, and then there is a space after the close parenthesis of the conditional, input arguments, etc. Maintaining this convention creates consistent code that is easy to review the content of without getting hung up on the formatting.
+
+Always end your file with a newline.
+
+    ```javascript
+      // good
+      if (x === 0) {
+        console.log('This is done correctly');
+      }
+
+      var timesTwo = function (x) {
+        return x * 2;
+      };
+
+      for (var i = 0; i < 10; i++) {
+        console.log('i is: ' + i);
+      }
+
+      // bad:
+      if(x !== 1){
+        console.log('This is not following our convention');
+      }
+
+      var timesThree = function(x){
+        return x * 3;
+      };
+    ```
+
+* You may use it to align two similar lines, but it is not recommended. This pattern usually leads to unnecessary edits of many lines in your code every time you change a variable name.
+
+    ```javascript
+      // discouraged:
+      var firstItem  = getFirst ();
+      var secondItem = getSecond();
+    ```
+
+* Put `else` and `else if` statements on the same line as the ending curly brace for the preceding `if` block
+    ```javascript
+      // good:
+      if (condition) {
+        response();
+      } else {
+        otherResponse();
+      }
+
+      // bad:
+      if (condition) {
+        response();
+      }
+      else {
+        otherResponse();
+      }
+    ```
+
+### Opening or closing too many blocks at once
+
+* The more blocks you open on a single line, the more your reader needs to remember about the context of what they are reading. Try to resolve your blocks early, and refactor. A good rule is to avoid closing more than two blocks on a single line--three in a pinch.
+
+    ```javascript
+      // avoid:
+      _.ajax(url, {success: function(){
+        // ...
+      }});
+
+      // prefer:
+      _.ajax(url, {
+        success: function () {
+          // ...
+        }
+      });
+    ```
+
+### Variable declaration
+
+* Use a new var statement for each line you declare a variable on.
+* Do not break variable declarations onto multiple lines.
+* Use a new line for each variable declaration.
+* See http://benalman.com/news/2012/05/multiple-var-statements-javascript/ for more details
+
+    ```javascript
+      // good:
+      var ape;
+      var bat;
+
+      // bad:
+      var cat,
+          dog
+
+      // use sparingly. Tux tends to use it specifically when declaring variables that are reused:
+      var eel, fly;
+    ```
+
+### Capital letters in variable names
+
+* TuxedoJS follows standard camelCase variable naming. If you are creating a constructor function that will be invoked using new, the variable name should
```

Comment:
Incomplete sentence.  Change to say ...constructor function that will be invoked using new or a similar object creation pattern, the first letter of the variable name should be capitalized as well.